src/js/hooks/cycleCount/useInventoryValidation.js

```diff
@@ -58,7 +58,10 @@ const useInventoryValidation = ({ tableData }) => {
         const expirationDate = item?.inventoryItem?.expirationDate;
         return expirationDate == null ? null : moment.utc(expirationDate);
       });
-      const uniqueDates = _.uniqWith(expirationDates, (arrVal, othVal) => arrVal?.isSame(othVal));
```

Comment:
This was returning null when expiration date is null, which is falsy, which uniqWith treats as "not the same", hence the error.